docs/quality-assurance/Web-Accessibility.md

```diff
@@ -0,0 +1,26 @@
+# Web Accessibility Testing
+
+Accessibility is not a 'nice to have', it's a legal requirement for companies and organisations to be inclusive for everyone.
+
+Accessibility Testing is not confirmation-based. Instead of testing the functionality works, testers need to look for how it **can't** be used because of an accessibility barrier.
+
+Automated tools include:
+
+- core installed in delivery pipeline - https://www.deque.com/axe/
+- Lighthouse - https://developers.google.com/web/tools/lighthouse
+
+Accessibility Testing is 'hands on'. While automated tests will catch things like missing alt text, contrast problems, and missing form labels; they are no substitute for manual hands on testing. Pre-written manual testing scripts cannot be relied upon because they provide no more insight than automated testing. The tester must explore different ways of using the application in order to tease out problems with accessibility.
+
+Manual Test Tools include:
+
+- aXe - https://www.deque.com/axe/
+- Wave - https://wave.webaim.org/extension/
+- Tenon - https://tenon.io/
+- Lighthouse - https://developers.google.com/web/tools/lighthouse
+- NVDA Screen Reader - https://www.nvaccess.org/
+
+For more information regarding accessibility requirements see:
+
+- Government Websites are subject to WCAG 2.1: https://www.w3.org/TR/WCAG21/
```

Comment:
This is missing the required rating, feels like it should say AA somewhere? The phrasing is also a little odd, it makes the ratings system sound like the legislation. Sticking Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 is probably overkill, though! :)

Also might be worth linking to https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag